### ISSUE TITLE
Requesting OS sim update.

### ISSUE THREAD

**Issue by Poffertjesporem** [FrSky-OS/discussion#67]
Please update the custom model creation section, then we can give better feedback to improve the software.

Adding the audio features would also be nice.

**Comment by MIKE-DAILEY** [comment]
The simulator "Info" screen shows that the simulator is FrTX version 1.1 and as been available for sometime now.  Considering there are now 58 important items posted here on GitHub, I'm hopeful that the simulator is lagging behind the actual Horus FrTX firmware. I hope!

**Comment by janekx** [comment]
Yes, it is little lagging. FrSky will release new Simu based on actual version ready for Horus. Date is still  not know.

**Comment by Poffertjesporem** [comment]
Can I have a link to the "info"?

**Comment by janekx** [comment]
Info for what ?

**Comment by Poffertjesporem** [comment]
I thought you ment the sim information from frsky, but its the info button lol.

**Comment by janekx** [comment]
I am in direct touch with FrSky quite often, it is based on our communication, it is not somewhere seen info :)

**Comment by Poffertjesporem** [comment]
Thanks for the clarification :)

**Comment by AloftHobbies** [comment]
We have both requested an update from FrSky and they should be taking an active roll here on github too.

**Comment by Poffertjesporem** [comment]
That's great news, please request a auto save function. Now after restart/crash/model change, setting changes go back to original state.

**Comment by Woita2095** [comment]
I use the Simu since today (I think it is ver. PCHoru1026)
My firs impression is concern about the Simu itself:
 
1. The color display is NOT touchscreen, is it? It is very tricky (sometimes almost impoossible) to move from one item to another, or change the value. Will the real Horus have a touchscreen?
I dont know what control element in simulator to use, whether cursor keys on my keyboard (sometimes works, sometimes not), or click mouse onto directional pad on the transmitter, or rotate, or what? Reactions and movements are unclear for me.

2. The window called "Control Pad": It should look like the control panel at the real transmitter, shouldn't it? I think the Stick J3 should be situated vertically, Stick J4 horizontally, orientation of the control elements in the correct direction, layout should be as on the real transmitter, etc. Please improve the graphic layout of the sim!

**Comment by janekx** [comment]
Hi Woita,  
Welcome here,  we know the simu is not  well for control but it is not issue to radio but some using  comfort.  We focusing on bugs system inself. 

Horus will not have touchscreen.  

As we told there will be  simu  update so we got better look at actual system status and can test and tune.

**Comment by MIKE-DAILEY** [comment]
"Horus will not have touchscreen."
For some reason I thought it did, but revisiting the published spec from November 23rd it only shows "Industrial high resolution IPS screen (480*272), readable outdoor"

**Comment by Poffertjesporem** [comment]
Please clarify on the touch screen.
A Quote from Aloft hobbys " I have some concerns with how the navigation buttons and touch screen are being used with this simulator" https://github.com/FrSky-OS/discussion/issues/5

**Comment by MIKE-DAILEY** [comment]
If the screen is not touch enabled with FrTX OS I think that would create more incentive to run OpenTX.

**Comment by janekx** [comment]
FrSkyOS and OpenTX,  both  are not  designed to touchscreen.  
Any way no latest news say about touchscreen.  
For me it is good,  no dirty lcd :)

**Comment by MIKE-DAILEY** [comment]
I guess when I saw "Industrial high resolution IPS screen (480*272), readable outdoor" I assumed a screen like that would have touch.  But the software development and mapping all those screens for touch is a huge effort.  

Like you say though touch screens around glow fuel would look messy pretty fast.

**Comment by Woita2095** [comment]
Can you imagine tablet without touchscreen, only with buttons?  Just try imagine such operating. It is not impossible, but uncomfortable... Tablet without touchscreen would be unsaleable.
Touching is very comfortable, quick and easy.
"Any way no latest news say about touchscreen."
That is pitty.
:-(

**Comment by janekx** [comment]
Touchscreen was more wish then reality.  
IPS screen is good but not perfect at all in direct sun.  Best way is Oled or Amoled where dots light self and add there backlight, but  TN or IPS still needs backlight and dots not illuminate self.  
Hope and wish will be LCD really well visible outdoor.

**Comment by hrenz** [comment]
No, latest Horus version has no touchscreen!
but a nice daylight clour display

With the driverfiles for Horus development  there are touch screen drivers included.

Does anyone have a later Horus simuversion as Simu1.1
I dont think so.

This Simu looks like a QT4 Programm GUI for an old openTx in the background

**Comment by Adelaxie** [comment]
We will update simu on frsky BBS,please tell me your username on frsky BBS  here.

**Comment by Woita2095** [comment]
Woita2095
Thanks!


Dne 30.12.2015 v 8:20 Adela napsal(a):

**Comment by Adelaxie** [comment]
Please check :BBS < FrSky Developing Union ‹ Project ‹ Limited Area ‹ Horus FrTX operation system simulator (beta test version)

**Comment by Poffertjesporem** [comment]
"
What is our company website domain name?:
This question is a means of preventing automated form submissions by spambots."

Can't create account due to the security question, please supply the domain name(frsky-rc.com does not work).

**Comment by Woita2095** [comment]
You have to write "www.frsky-rc.com" :-) That's little bit tricky...

**Comment by Poffertjesporem** [comment]
The 3th attempt it did,thanks.

**Comment by janekx** [comment]
Redownload the GUI 1230 there is new version !!!

**Comment by MIKE-DAILEY** [comment]
GPS!!

**Comment by AloftHobbies** [comment]
See this post for a new link for the sim download. Easier and faster than getting it from the FrSky Forum.
click here --> https://github.com/FrSky-OS/discussion/issues/4

**Comment by AloftHobbies** [comment]
Also, we will be making this github site private very soon. You may need to request to become a member after we do this. Sorry for the trouble. - Wayne

**Comment by Poffertjesporem** [comment]
Thanks!
